docs/types/dynamic-dispatch.md

```diff
@@ -227,68 +257,64 @@ the `Receiver`, and then finds the method definition in its _symbol table_:
   `symbol` is defined in any of the modules, select it and stop.
 - Raise `Name_Not_Found` panic and stop.
 
-## Method evaluation
-
-This section describes the _method evaluation_ process. We assume that a
-particular method definition was already selected by the
-[method resolution](#method-resolution) process.
-
-Let's have `method` be a method definition resolved from
-[Method Resolution](#method-resolution) process, that is defined as
-`method [self] [parName=[defaultValue]]* = <body expression>`.
+## Examples
 
-If `self` parameter is specified, we call it an _instance method_, otherwise, we
-call it a _static method_.
+```
+@Builtin_Type
+type Any
+    to_text self = "???"
 
-### Instance method call evaluation
+type My_Type
+    Cons
+    method self x = x + 1
 
-The method call expression in the format of `obj.method [[argName=]argValue]*`,
-where `obj` is an instance (value / atom) of type `T`, and `method` is an
-_instance method_ with `self` argument defined either on `T` or outside `T` as
-an _extension method_, the method evaluation process is as follows:
+obj = My_Type.Cons
+```
 
-- `self` argument cannot be specified explicitly
-- Go to [Arguments evaluation](#argument-evaluation) to evaluate `argValue`s and
-  bind them to `argName`s.
+### Example (a)
```

Comment:
- Instead of examples in the documentation, put them into a test case and link from here. Maybe?
- Documentation full of examples has only one future: to get outdated.

Reply:
Valid point. I will definitely add such tests in https://github.com/enso-org/enso/pull/13962. This whole documentation is, in fact, the **desired behavior** that is being implemented in #13962.

Reply:
Added as a checklist point in #13962.